.ci/util.sh

```diff
@@ -16,9 +16,9 @@ function checkForVariable() {
 
 function getMavenProperty {
   property="\${$1}"
-  echo "$(mvn -e --no-transfer-progress -q -Dexec.executable='echo' \
+   mvn -e --no-transfer-progress -q -Dexec.executable='echo' \
```

Comment:
ALso, indentation is incorrect. -1 space.

Reply:
Done @rnveach .